cita-network/src/node_manager.rs

```diff
@@ -794,3 +876,18 @@ impl GetPeerCountReq {
         }
     }
 }
+
+pub struct ConnectedSelfReq {
+    addr: SocketAddr,
+}
+
+impl ConnectedSelfReq {
+    pub fn new(addr: SocketAddr) -> Self {
+        ConnectedSelfReq { addr }
+    }
+
+    pub fn handle(self, service: &mut NodesManager) {
+        service.self_addr = Some(self.addr);
```

Comment:
If you listen to multiple addresses or are mapped to multiple addresses, the processing here is problematic.

Can be modified later, CITA do not currently support multi-address listening

Reply:
OK